README.md

```diff
@@ -18,6 +18,16 @@ A Secret of Mana Randomizer bot for Discord.
 - .Net Core: https://dotnet.microsoft.com/download
 - DSharpPlus: https://github.com/DSharpPlus/DSharpPlus
 
+### Permissions
+
+For all commands to work, the following permissions must be given to the bot on discord
+
+- Send Messages
+- Manage Messages
+- Manage Roles
+- Access Channels
+- Add Reactions
```

Comment:
Using the `SendSuccessReaction` functions, we're dynamically checking for reaction permissions, so that you don't need to give the bot the reaction permission.
If we flat-out require adding reactions, we might want to remove the permission check in the helper functions to improve performance.

Reply:
True, we could keep this a soft requirement...
I started adding `[RequirePermissions]` attributes because we are at a point where the bot won't work as it should unless it has these permissions enabled, but adding reactions could remain a soft-requirement.